apps/system/lockscreen/js/state/lockscreen_basic_state.js

```diff
@@ -57,6 +61,17 @@
    * Or the component cannot track the last active state.
    */
   LockScreenBasicState.prototype.transferTo = function() {
+    if (this._transferred) {
+      this.logger.debug('Prevent transferring racing');
+      var nullifized = new Stream();
+      // This would return a process could be concated but would do nothing.
+      // It's better to formally provide a API from Process, like
+      // Process.maybe() or Process#nullize(), but this is a simplier solution.
+      return nullifized.start().next(() => nullifized.stop());
```

Comment:
This function would return a Process/Stream could be concated with 'next' since the nature of the `stop` and `start` method. So if I return a Promise this would probably break unknown user code, and add the extra burden of detecting this un-uniform type on each caller.